src/Lexer.php

```diff
@@ -398,62 +398,55 @@ private function stripcslashes(string $str, string $quoteType): string
             'v' => "\v",
         ];
```

Comment:
`$specialChars` can probably be moved to a private const?

Reply:
Done